pwnlib/tubes/ssh.py

```diff
@@ -553,7 +553,7 @@ class ssh(Timeout, Logger):
 
     def __init__(self, user=None, host=None, port=22, password=None, key=None,
                  keyfile=None, proxy_command=None, proxy_sock=None,
-                 level=None, cache=True, ssh_agent=False, *a, **kw):
+                 level=None, cache=True, ssh_agent=False, banner=True, *a, **kw):
```

Comment:
I would prefer the name is `checksec` vs `banner`

Reply:
My first approach as well, but changed my mind when I was looking at the 'checksec()'  function itself, and saw 'banner' as argument. Maybe should do a cleanup too, as it seems to be an artifact of some kind?

Reply:
`checksec` shouldn't collide with `self.checksec`,  it should be fine